wp/resources/sample_binaries/cbat-multicompiler-samples/csmith/run_wp.sh

```diff
@@ -0,0 +1,17 @@
+set -x
+
+dummy_dir=../../dummy
+
+compile () {
+  make
+}
+
+run () {
+  bap $dummy_dir/hello_world.out --pass=wp \
+    --wp-compare=true \
+    --wp-inline \
```

Comment:
We changed how the inline flag works. To inline everything, we should `--wp-inline=.*`

Reply:
Oh right. The example crashes regardless, I believe.

Reply:
But I'll fix it.

Reply:
Sounds good!

Reply:
All right, fixed.